example/cumulus-tf/outputs.tf

```diff
@@ -6,6 +6,8 @@ output "archive_api_redirect_uri" {
   value = module.cumulus.archive_api_redirect_uri
 }
 
+# TEA-Specific outputs
+
```

Comment:
Note that these are not optional. We're always deploying `thin_egress_app` with this configuration. It doesn't get switched off when using cumulus_distribution. We just deploy both.

Reply:
Separate comment: can we change these output names? Otherwise it's a bit confusing which output is for TEA vs cumulus distribution

`distribution_api_uri` -> `cumulus_distribution_api_uri`
`distribution_api_redirect_uri` -> `cumulus_distribution_api_redirect_uri`

I might even go all the way and do these all well:

`distribution_url` -> `tea_distribution_url`
`distribution_redirect_uri` -> `tea_distribution_redirect_uri`
`s3_credentials_redirect_uri` -> `tea_s3_credentials_redirect_uri`

Since these outputs are from `example/cumulus-tf`, they're not user facing so it should be safe to change them. It just helps avoid confusion for us

Reply:
Oh yeah, good point. I guess these really aren't used elsewhere other than the console output on a deploy.

https://github.com/nasa/cumulus/pull/2300/commits/be47b39f61c79ac38d4323f7e778168ece608501

Reply:
I think it's better to change the input variable `distribution_url -> tea_distribution_url` as well if you change the output.
https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/variables.tf#L178
https://github.com/nasa/cumulus/blob/791a955480b12faa1d511c4d7abcdfae0f177124/example/cumulus-tf/thin_egress_app.tf#L41

Reply:
Yup! Good idea. Here's the update:

https://github.com/nasa/cumulus/pull/2300/commits/10d32874983563d5b405b21586555b8eb30f025f

I also added `tea_distribution_url` to the tfvars.example:

https://github.com/nasa/cumulus/pull/2300/commits/00c539d3e79b7f8c1e0ea123ad137ba77d1afa1e

Do my comments there make sense? Do you always need to uncomment and use one those variables? Is there a situation where you actually don't need to overwrite the URL to add a port?

Reply:
Both `tea_distribution_url` and `cumulus_distribution_url` are optional.  You don't have to overwrite in the case the original url is accessible.